components/eamxx/src/physics/rrtmgp/scream_rrtmgp_interface.cpp

```diff
@@ -760,6 +820,9 @@ namespace scream {
             // Combine gas and cloud optics
             clouds_day.delta_scale();
             clouds_day.increment(optics);
+            if (extra_clnsky_diag) {
```

Comment:
I would move this to the extra_clnsky_diag logical chunk below to keep everything related to the extra_cldsky_diag call together so it's more obvious what goes into that calculation.